src/app/server/CommissioningWindowManager.cpp

```diff
@@ -286,6 +292,11 @@ CHIP_ERROR CommissioningWindowManager::StopAdvertisement()
     mPairingSession.Clear();
 
     mCommissioningWindowOpen = false;
+    mServer->GetExchangeManager().SetActiveStateForced(false);
+
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+    DeviceLayer::ConnectivityMgr().SetSEDPollingMode(DeviceLayer::ConnectivityManager::SEDPollingMode::Idle);
```

Comment:
This is not right.  There's an outstanding spec issue about this too...

There are multiple things in the spec that all require fast-poll mode and could be happening in parallel.  We need to stay in fast-poll mode as long as at least one of those things is going on, not exit it as soon as any of them completes.

The right API for this is probably some API that lets you increment/decrement a counter of fast-poll-consumers that are active and automatically enters Active/Idle mode when the counter transitions from/to 0 respectively.

Reply:
Yes, I agree with it. Thank you for that input and API suggestion that is very nice. I modified the code, so the `RequestSEDFastPollingMode` method is available for external modules and handles mentioned counter. Old `SetSEDPollingMode` method is now private and called depending on the counter state.